src/main/java/ar/edu/ungs/fleet_manager/controls/domain/Control.java

```diff
@@ -0,0 +1,146 @@
+package ar.edu.ungs.fleet_manager.controls.domain;
+
+import ar.edu.ungs.fleet_manager.users.domain.UserId;
+import ar.edu.ungs.fleet_manager.vehicles.domain.VehicleId;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.Objects;
+import java.util.UUID;
+
+public final class Control {
+    private final ControlId id;
+    private ControlType type;
+    private ControlSubject subject;
+    private ControlDescription description;
+    private VehicleId idVehicle;
+    private ControlPriority priority;
+    private final LocalDateTime date;
+    private ControlStatus status;
+    private ControlAssigned assigned;
```

Comment:
Veo todo okey! Lo que faltaría que es UserId relacionando al mecánico


```suggestion
public final class Control {
    private final ControlId id;
    private ControlType type;
    private ControlSubject subject;
    private ControlDescription description;
    private VehicleId idVehicle;
    private ControlPriority priority;
    private final LocalDateTime date;
    private ControlStatus status;
    private UserId operatorId;
```